modules/userId/index.js

```diff
@@ -482,5 +485,6 @@ init(config);
 // add submodules after init has been called
 attachIdSystem(pubCommonIdSubmodule);
 attachIdSystem(unifiedIdSubmodule);
+attachIdSystem(liveIntentIdSubmodule);
```

Comment:
The liveIntentIdSubmodule cannot be auto included. Only `unifiedId` and `pubCommonId` are currently allowed to be auto-included, and in Prebid 3.0 event pubCommonId and unifiedId will no longer be auto-included.

Reply:
Hi @idettman, then it should be extracted out of `modules/userId` structure into `modules/liveIntentIdSystem` so it can be added to the list of modules during build?

Reply:
Check how the id5 submodule is implemented, basically remove  changes to index.js, and add to the .submodules.json. As long as defined in the submodules.json, it will be added when included in the modules list

Reply:
@idettman i think this has been resolved with the last commit